openmrs/api/src/main/java/org/projectbuendia/openmrs/api/ProjectBuendiaService.java

```diff
@@ -26,5 +31,13 @@
  */
 @Transactional
 public interface ProjectBuendiaService extends OpenmrsService {
-    /* Add service methods here */
+
+    /** Sets the DAO for this service. This is done by DI and Spring. */
+    void setDAO(ProjectBuendiaDAO dao);
+
+    /**
+     * Returns all encounters modified on or after the given {code date}.
+     * @param date if {@code null}, returns all encounters since the beginning of time.
+     */
+    List<Encounter> getEncountersModifiedOnOrAfter(Date date);
```

Comment:
Maybe! But we can do that later, and possibly at a higher level of the API than the DB layer.